components/Tabs/src/Theming/MDCTabs+MaterialTheming.m

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
This file needs to be named "MDCTabBar+MaterialTheming.m" and so does the header.  See go/mdc-ios-theming-extensions for naming conventions.

> MDC<#ThemableClass#>+MaterialTheming.h

Reply:
Addressed in 4e8cb3cc3ec6eb9a0b5536dac519b6b866791415